.env.local.example

```
@@ -31,50 +31,50 @@ VORTEX_PROVISION_OVERRIDE_DB=1
-# Always override existing downloaded DB dump.
+# Always override existing fetched DB dump.
 VORTEX_FETCH_DB_FORCE=1
 
-#;< DB_DOWNLOAD_SOURCE_FTP
+#;< DB_FETCH_SOURCE_FTP
 # Database dump file sourced from FTP.
 
 # Database dump FTP user.
 VORTEX_FETCH_DB_FTP_USER=
 # Database dump FTP password.
 VORTEX_FETCH_DB_FTP_PASS=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VORTEX_FETCH_DB_FTP_PASS key should go before the VORTEX_FETCH_DB_FTP_USER key

(UnorderedKey)